### DISCUSSION TITLE
The Imitation Game

### DISCUSSION THREAD

**Discussion by giscus** [gxanshu/gxanshu-site#103] [Q&A]
# The Imitation Game

my learning from Alan turing's life and how he becomes a great inspiration for me.

https://gxanshu.in/blog/the-imitation-game/

**Comment by gxanshu** [comment]
Initially i came to think of if Alan turing created the first computer then why we calls **Charles Babbage** as father of computer.

Charles Babbage is often called the "Father of the Computer" because of his early conceptual work on mechanical computing devices, while Alan Turing is considered the "Father of Modern Computer Science" because of his theoretical contributions to the field, which influenced the design and development of electronic computers.
